tests/lint/test_pylinter.py

```diff
@@ -41,3 +42,19 @@ def test_check_deprecation(linter: PyLinter, recwarn):
     linter.check("myfile.py")
     msg = recwarn.pop()
     assert "check function will only accept sequence" in str(msg)
+
+
+def test_crash_during_linting(
+    linter: PyLinter, capsys: CaptureFixture[str], tmpdir: LocalPath
+) -> None:
+    with mock.patch(
+        "pylint.lint.PyLinter.check_astroid_module", side_effect=RuntimeError
+    ):
+        linter.crash_file_path = str(tmpdir / "pylint-crash-%Y")
+        linter.check([__file__])
+        out, err = capsys.readouterr()
+        assert not out
+        assert not err
+        files = tmpdir.listdir()
+        assert len(files) == 1
+        assert "pylint-crash-20" in str(files[0])
```

Comment:
Will python/pylint still be relevant in 2100 ? If so we're creating work for the next generation of maintainers 😄

Reply:
Copied from the test above 😄